scripts/docker_devboot.sh

```diff
@@ -71,6 +71,8 @@ run_master() {
 CFG_SITE_URL = u'http://localhost:5000'
 CFG_SITE_SECURE_URL = u'http://localhost:5000'
 CFG_REDIS_HOSTS = {'default': [{'db': 0, 'host': 'cache', 'port': 6379}]}
+CFG_DATABASE_NAME = u'invenio'
```

Comment:
If they do they are on their own. Probably they are going to use external database anyway so the docker will just complicate their lives.